meta-lmp-base/recipes-core/images/initramfs-ostree-lmp-recovery.bb

```diff
@@ -9,21 +9,10 @@ SRC_URI = "\
 "
```

Comment:
```
SRC_URI = "\
	file://uboot_env.sh \
	file://udhcpc.sh \
	file://image_download.sh \
"
```
This files in the `SRC_URI` block at https://github.com/foundriesio/meta-lmp/pull/845/files#diff-d95f93050a15afee560c55e19124449b4df89d7a90def8300b579da9533071c2L5-L9 is been moved  from `meta-lmp-base/recipes-core/images/initramfs-ostree-lmp-recovery` to `meta-lmp-base/recipes-core/initrdscripts/lmp-recovery-framework` so they don't found any more on the previous location as they are now on https://github.com/foundriesio/meta-lmp/pull/845/files#diff-bb5149a6d934f9661851d89d0842ca6921bdc65491b201e832d34e40fe42f262R7-R10